app/views/layouts/application.html.erb

```diff
@@ -7,6 +7,7 @@
     <%= stylesheet_link_tag 'application', media: 'all' %>
     <%= javascript_include_tag 'application' %>
     <%= csrf_meta_tags %>
+    <link href='https://fonts.googleapis.com/css?family=Source+Sans+Pro:400,200,200italic,300,300italic,400italic,600' rel='stylesheet' type='text/css'>
```

Comment:
@drownedout Can we get this moved into the asset pipeline rather than linking to it directly? Would prefer that this live in, and get loaded from, `vendor/assets/stylesheets` rather than a live http link.